x/interchainquery/keeper/msg_submit_query_response_test.go

```diff
@@ -38,20 +43,19 @@ func (s *KeeperTestSuite) SetupMsgSubmitQueryResponse() MsgSubmitQueryResponseTe
 	query := types.Query{
 		Id:           expectedId,
 		ConnectionId: s.TransferPath.EndpointA.ConnectionID,
-		ChainId:      hostChainId,
+		ChainId:      HostChainId,
 		QueryType:    types.BANK_STORE_QUERY_WITH_PROOF,
-		Request:      append(data, []byte(hostChainId)...),
+		Request:      append(data, []byte(HostChainId)...),
 		Period:       sdk.NewInt(0),
 		LastHeight:   sdk.NewInt(height),
 		CallbackId:   "withdrawalbalance",
-		// Ttl:          uint64(2545444772), // August, 2050,
-		Ttl:    uint64(1), // August, 2050,
-		Height: height,
+		Ttl:          uint64(12545592938) * uint64(1000000000), // set ttl to August 2050, mult by nano conversion factor
```

Comment:
Since the TTL is checked using the block time (which is hard coded), you can set this relative to that.

ex:
```
uint64(s.Coordinator.CurrentTime.UnixNano() + 60_000_000_000)
```

It's generally good to avoid using real timestamps in tests, even if you set it crazy far into the future like this.